robot/description/urdf/constants.xacro

```diff
@@ -18,14 +21,67 @@
     <xacro:property name="caster_link_to_wheel_dist_z" value="-0.04361749"/>
 
     <!-- Wheels -->
+    <!-- TODO: update parameters once they become known -->
     <xacro:property name="wheel_vertical_offset" value="-0.61759696"/>
     
+    <xacro:property name="wheel_left_joint" value="left_wheel_joint"/>
+    <xacro:property name="wheel_right_joint" value="right_wheel_joint"/>
+
     <xacro:property name="wheels_separation" value="0.875"/>
     <xacro:property name="wheel_diameter" value="0.25"/>
 
+    <xacro:property name="wheel_acceleration" value="1.0"/>
+    <xacro:property name="wheel_torque" value="20"/>
+
     <xacro:property name="wheel_mass" value="4.45"/>
 
     <!-- Gazebo Motor -->
+    <!-- TODO: update parameters once they become known -->
+
     <xacro:property name="diff_drive_update_rate" value="30.0"/>
+
+    <xacro:property name="diff_drive_commandTopic" value="cmd_vel"/>
+    <xacro:property name="diff_drive_odometryTopic" value="odom"/>
+
+    <xacro:property name="diff_drive_odometryFrame" value="odom"/>
+    <xacro:property name="diff_drive_robotBaseFrame" value="base_link"/>
+
+    <xacro:property name="diff_drive_odometrySource" value="encoder"/>
+
+    <xacro:property name="diff_drive_publishWheelTF" value="false"/>
+    <xacro:property name="diff_drive_publishOdomTF" value="true"/>
+    <xacro:property name="diff_drive_publishTf" value="true"/>
+    <xacro:property name="diff_drive_publishWheelJointState" value="true"/>
+
+    <xacro:property name="diff_drive_legacymode" value="false"/>
+    <xacro:property name="diff_drive_rosDebugLevel" value="na"/>
+
+    <!-- Constants for hokuyo 2D LIDAR sensor -->
+    <xacro:property name="laser_visualize" value="false"/>
+    <xacro:property name="laser_update_rate" value="30"/>
```

Comment:
This refresh rate seems a little off (just a gut feeling). Looking [here](https://www.roscomponents.com/en/lidar-laser-scanner/84-urg-04lx.html) the refresh rate is only 10 Hz. I suggest reviewing that spec sheet and a few others. I don't rmb our exact Hokuyo but it looks like the one in the link just posted - ours is probably just a slightly older version